app/forms/application_form.rb

```
@@ -23,7 +23,7 @@ def self.immutable_attributes
   # @param deposit [Boolean] whether the form is being used for a deposit
   def valid?(deposit: false)
     @deposit = deposit
-    super()
```

It's fine to let the args bubble up, apparently.